src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -1476,6 +1477,42 @@ def prepare_runner(fuzzer_path,
   return runner
 
 
+def minimize_corpus(target_path, arguments, input_dirs, output_dir,
+                    reproducers_dir, max_time):
+  """Minimizes the corpora in |input_dirs| and writes the resulting corpus to
+  |output_dir|."""
+  del arguments
+  del reproducers_dir
+
+  set_additional_sanitizer_options_for_afl_fuzz()
+
+  # AFL minimizes corpora using the afl-cmin tool.
+  afl_tools_path = os.path.dirname(target_path)
+  afl_cmin_path = os.path.join(afl_tools_path, 'afl-cmin')
+  afl_cmin_runner = new_process.UnicodeProcessRunner(afl_cmin_path)
+
+  os.chmod(target_path, 0o775)
+  os.chmod(afl_cmin_path, 0o775)
+
+  # afl-cmin only works on one directory, copy all inputs to a single directory.
+  with tempfile.TemporaryDirectory as combined_input_dir:
```

Comment:
does this not need `tempfile.TemporaryDirectory()` (the parentheses)?

Reply:
Yup